torch_geometric/loader/neighbor_loader.py

```diff
@@ -159,8 +159,12 @@ def _sparse_neighbor_sample(self, index: Tensor):
         )
         return node, row, col, edge
 
-    def _hetero_sparse_neighbor_sample(self, index_dict: Dict[str, Tensor]):
-        if self.node_time_dict is None:
+    def _hetero_sparse_neighbor_sample(self, index_dict: Dict[str, Tensor],
```

Comment:
What we could do instead is define this as
```suggestion
    def _hetero_sparse_neighbor_sample(self, index_dict: Dict[str, Tensor], **kwargs):
```
and then later do
```python
kwargs.get('num_neighbors', self.num_neighbors),
kwargs.get('node_time_dict', self.node_time_dict),
...
```